src/components/WriteButton/WriteButton.js

```diff
@@ -0,0 +1,12 @@
+import StyledWriteButton, { StyledAddImg } from "./styled";
+
+const WriteButton = (props) => {
+  return (
+    <StyledWriteButton onClick={props.onClick}>
+      <StyledAddImg src="add.svg" alt="add"></StyledAddImg>
```

Comment:
When dealing with svg file I prerer to import them.
```jsx
import AddIcon from "../icons/add.svg";

<StyledAddImage src={AddIcon}  alt="add" />
```

Another thing that I did was:
When you don't pass a child to a component you can use the shorthand version `<Name ...props />`